presto-main/src/main/java/com/facebook/presto/operator/SortedPositionLinks.java

```diff
@@ -211,17 +210,35 @@ public int next(int position, int probePosition, Page allProbeChannelsPage)
             return -1;
         }
         // break a position links chain if next position should be filtered out
-        if (applyLessThanFunction(nextPosition, probePosition, allProbeChannelsPage)) {
-            return nextPosition;
+        int count = inequalityJoinFilterConjuncts.size();
```

Comment:
It is more readable to use `for (var: Itarable)` syntax instead `count` variable.

```
for (JoinFilterFunction conjunct : inequalityJoinFilterConjuncts) {
 ....
```